guide/command-handling/README.md

```diff
@@ -81,7 +81,7 @@ const client = new Discord.Client();
 :::
 
 ::: tip
-If you aren't exactly sure what Collections are, they're a class that extend JS's native Map class and include more extensive, useful functionality. You can read about Maps [here](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Map), and see all the available Collection methods [here](https://discord.js.org/#/docs/main/stable/class/Collection).
+If you aren't exactly sure what Collections are, they're a class that extend JS's native Map class and include more extensive, useful functionality. You can read about Maps [here](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Map), and see all the available Collection methods [here](https://discord.js.org/#/docs/main/11.5.1/class/Collection).
```

Comment:
needs version specific docs link

Reply:
I mentioned that as no documentation for Collections on master, as they have separate category in docs now.
Thing is, the collection category is not really "working", as mentioned in https://github.com/discordjs/website/issues/41. I had to literally nuke whole local data for the website to be able to even see that category.

Reply:
I don't think this is going to be addressed in a timely manner (if at all), but the Link should still be correct for v12. If you think it's an issue i could see linking to <https://discord.js.org/#/docs/collection/master/class/Collection> instead, as the separate package approach to all things Collection is here to stay.